### ISSUE TITLE
Application homepage change

### ISSUE THREAD

**Issue by Ahmetto7** [HemantKArya/BloomeeTunes#81]
There is nothing visible on the application homepage. A feature can be added to the settings and the homepage can show services such as YouTube music. Here is the music service homepage.

**Comment by HemantKArya** [comment]
Yeah btw right now we already show the homepage from youtube/music endpoint. but still for specific other music homepages that can be good idea and wil make sure to work that in future.

**Comment by HemantKArya** [comment]
you can now change or remove home page by installing different plugins or removing them
